app/kubemci/pkg/gcp/urlmap/urlmapsyncer.go

```diff
@@ -219,6 +220,7 @@ func (s *URLMapSyncer) ingToURLMap(ing *v1beta1.Ingress, beMap backendservice.Ba
 	}
 	defaultBackend, beErr := getBackendService(ing.Spec.Backend, ing.Namespace, beMap)
 	if beErr != nil {
+		fmt.Printf("getBackendService error: %v", beErr)
```

Comment:
Its not clear to user what getBackendService is.
Change to ("error in getting backend service %s", name)?

Reply:
done.